nipype/algorithms/confounds.py

```diff
@@ -106,16 +270,121 @@ def _run_interface(self, runtime):
                         format=self.inputs.out_figure[-3:],
                         bbox_inches='tight')
             fig.clf()
+
         return runtime
 
     def _list_outputs(self):
         return self._results
 
 
+def compute_dvars(in_file, in_mask, remove_zerovariance=False):
+    """
+    Compute the :abbr:`DVARS (D referring to temporal
+    derivative of timecourses, VARS referring to RMS variance over voxels)`
+    [Power2012]_.
+
+    Particularly, the *standardized* :abbr:`DVARS (D referring to temporal
+    derivative of timecourses, VARS referring to RMS variance over voxels)`
+    [Nichols2013]_ are computed.
+
+    .. [Nichols2013] Nichols T, `Notes on creating a standardized version of
+         DVARS <http://www2.warwick.ac.uk/fac/sci/statistics/staff/academic-\
+research/nichols/scripts/fsl/standardizeddvars.pdf>`_, 2013.
+
+    .. note:: Implementation details
+
+      Uses the implementation of the `Yule-Walker equations
+      from nitime
+      <http://nipy.org/nitime/api/generated/nitime.algorithms.autoregressive.html\
+#nitime.algorithms.autoregressive.AR_est_YW>`_
+      for the :abbr:`AR (auto-regressive)` filtering of the fMRI signal.
+
+    :param numpy.ndarray func: functional data, after head-motion-correction.
+    :param numpy.ndarray mask: a 3D mask of the brain
+    :param bool output_all: write out all dvars
+    :param str out_file: a path to which the standardized dvars should be saved.
+    :return: the standardized DVARS
+
+    """
+    import os.path as op
+    import numpy as np
+    import nibabel as nb
+    from nitime.algorithms import AR_est_YW
+
+    func = nb.load(in_file).get_data().astype(np.float32)
+    mask = nb.load(in_mask).get_data().astype(np.uint8)
+
+    if len(func.shape) != 4:
+        raise RuntimeError(
+            "Input fMRI dataset should be 4-dimensional")
+
+    # Robust standard deviation
+    func_sd = (np.percentile(func, 75, axis=3) -
+               np.percentile(func, 25, axis=3)) / 1.349
+    func_sd[mask <= 0] = 0
```

Comment:
Robust standard deviation sounds useful--we should consider moving it to another module so everyone can use it.